### DISCUSSION TITLE
interpolate negative array

### DISCUSSION THREAD

**Discussion by marioepsley** [cables-gl/cables_docs#231] [! Welcome - readme first!]
Might of found a bug.
https://dev.cables.gl/p/62704159d2e9455c92d40f45?s=Ka8M4oerrPfsurjz

trying to interpolate between these split number arrays and getting some weird results. I can't seem to smoothly transition between these coordinates. 

<img width="771" alt="Screenshot 2022-05-02 at 21 39 00" src="https://user-images.githubusercontent.com/36418408/166324384-394d0bf2-27dc-4a54-9f9a-3625ab5743cb.png">

**Comment by pandrr** [comment]
thanks,

- added a toggle for split to force numbers (they were strings)
- vizArrayTable now shows quotation marks to indicate that the array consists of strings
